components/org.wso2.carbon.identity.sts.passive/src/main/java/org/wso2/carbon/identity/sts/passive/custom/provider/CustomCryptoProvider.java

```diff
@@ -70,31 +72,16 @@ public CustomCryptoProvider(Properties properties, ClassLoader loader,
     protected KeyStore load(InputStream input, String storepass, String provider, String type)
             throws WSSecurityException {
 
-        KeyStore keyStore;
-
-        String tenantId = this.properties.getProperty(TENANT_ID_PROP);
-        String keyStoreName = this.properties.getProperty(KEY_STORE_NAME_PROP);
+        if (log.isDebugEnabled()) {
+            log.debug("Loading keystore...");
```

Comment:
Since we are only logging a plain string, we don't need to use the `log.isDebugEnabled()` check. We only need that check if the log message is costly.

Reply:
Fixed in https://github.com/wso2-extensions/identity-inbound-auth-sts/pull/160/commits/de068a87b5ff4a99e3334d601045abd23b766a96